### PR TITLE
Change favicon, title, and mission statement

### PR DESCRIPTION
**Pull request by isabelsang**
image

### CODE REVIEW

File: frontend/public/index.html [hunk 1]
```diff
@@ -2,19 +2,19 @@
...
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/lightbulb.ico?v=2" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
       content="Web site created using create-react-app"
     />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+<!--    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />-->
```
Comment (indhu-gunda) [inline frontend/public/index.html:12]: why are these getting commented?

Comment (isabelsang) [reply frontend/public/index.html:12]: When the favicon wasn't updating at first I commented them to see if they were what causing the issue and I forgot about it.  I honestly am not sure what they do.  Do you think I should just delete them?  The href they're referring to is the react logo/favicon

Comment (indhu-gunda) [reply frontend/public/index.html:12]: should they be changes to our icon?

Comment (indhu-gunda) [reply frontend/public/index.html:12]: looks like they have both the favicon and png format of the image in the public folder.

Comment (indhu-gunda) [reply frontend/public/index.html:12]: ooh I see!

**Pull request merged**